homeassistant/components/nut/__init__.py

```diff
@@ -166,6 +167,11 @@ def _model_from_status(status: dict[str, str]) -> str | None:
     )
 
 
+def _model_id_from_status(status: dict[str, str]) -> str | None:
+    """Find the best model ID value from the status."""
+    return status.get("device.part")
```

Comment:
The docstring seems copied from model. But the code is not doing any logic.

```suggestion
    """Return the device part as model ID, if available."""
    return status.get("device.part")
```

Reply:
Suggested docstring applied.  Thanks!